README.md

```diff
@@ -1,2 +1,50 @@
-# http-method-override
+# http-method-override [![Build Status](https://travis-ci.org/rstgroup/http-method-override.svg?branch=master)](https://travis-ci.org/rstgroup/http-method-override)
 HTTP method override service
+
+Library allow to override method by header. Why? Because some clients cannot send custom method. It can be used as
```

Comment:
This library allows you to override HTTP method using HTTP header ie. `X-HTTP-Method-Override`. Why? Because some clients cannot send non-standard methods like ie. `LINK`.